suite_cam6.xml

```diff
@@ -0,0 +1,372 @@
+          </subcycle>
+          <scheme>aero_tracer_indx</scheme>           <!-- Determine aerosol tracer index (managed by registry/framework?) -->
+          <scheme>depvel_m_to_pa</scheme>             <!-- Convert the deposition velocity from m/s to Pa/s, and write to history file (maanged by registry/framework?) -->
+          <scheme>dust_sediment_tend</scheme>         <!-- Calculate host model tendencies and surface fluxes from dry deposition -->
+          <scheme>drydep_diag_output</scheme>         <!-- Calculate dry deposition diagnostic variables, and output them to history file -->
+        </subcycle>
+      </subcycle>
+      <scheme>set_srf_drydep</scheme>                 <!-- Add dry deposition surface fluxes to host model, if aerodep_flx_prescribed is False -->
+      <!-- **************************** -->
+      <scheme>charge_balance</scheme>                 <!-- Enforce charge nuetrality, if chemistry simulates electrons -->
+      <!-- Gravit wave tendencies -->
+      <!-- **************************** -->
+      <!-- This interface depends heavily
+           on namelist inputs, with entire
+           blocks of code being used (or not)
+           depending on said namelist choices.
+           However, the general structure of these
+           code blocks is more or less the same,
+           so a "generic" scheme set is included
+           here, which will need to be expanded
+           out once "if-statement" like dependencies
+           can be added to suite definition files
+           like this one -->
+      <scheme>set_dry_to_wet</scheme>                  <!-- Convert mixing ratios from dry air to air + water vapor -->
+      <scheme>gw_prof</scheme>                         <!-- Calculate layer interface densities and Brunt-Viasala frequencies (combine with other schemes?) -->
+      <!-- Generic gravity wave scheme order  -->
+      <!-- +++++++++++++++++++++++++ -->
+      <scheme>effgw_calc</scheme>                      <!-- Calculate the efficiency of gravity wave momentum transfer -->
+      <scheme>gw_src_calc</scheme>                     <!-- Determine gravity wave sources -->
+      <scheme>gw_drag_prof</scheme>                    <!-- Solve for the drag profile with the gravity wave spectrum -->
+      <scheme>calc_taucd</scheme>                      <!-- Project stress into directional components -->
+      <scheme>egwdffi_tot_calc</scheme>                <!-- add diffusion coefficients (manageable by registry/framework?) -->
+      <scheme>momentum_flux_calc</scheme>              <!-- Calculate momentum fluxes/wind tendencies -->
+      <scheme>energy_change</scheme>                   <!-- Calculate atmospheric energy change -->
+      <scheme>energy_fixer</scheme>                    <!-- Ensure energy conservation -->
+      <scheme>gw_cam_update</scheme>                   <!-- Placeholder for CAM gravity wave ptend and diagnostic output calculations -->
```

Comment:
Please remove the ptend part of these comments.

Reply:
I've now removed any reference to "ptend" from the file, with the one exception being the `co2_cycle_set_ptend` scheme name, given that I wanted to keep the name of the scheme the same as the actual subroutine in CAM.  However, I imagine this can be changed in the future.